lib/hexpm/repository/package.ex

```diff
@@ -9,6 +9,7 @@ defmodule Hexpm.Repository.Package do
     field :name, :string
     field :docs_updated_at, :utc_datetime_usec
     field :latest_version, Hexpm.Version, virtual: true
```

Comment:
with this patch we can remove latest_version in favour of the new latest_release. which means in the other scenarios we'd fetch a bit more data from the db that we don't need but that's most likely fine. wdyt?